src/devhub/notification/Left.jsx

```diff
@@ -16,18 +17,33 @@ return props.type ? (
       : type == "mention"
       ? "mentioned you in their"
       : type}
-    <a
-      className="fw-bold text-muted"
-      href={href({
-        widgetSrc: "${REPL_DEVHUB}/widget/app",
-        params: {
-          page: "proposal",
-          id: props.proposal,
-        },
-      })}
-    >
-      DevHub proposal
-    </a>
+    {props.proposal ? (
+      <a
+        className="fw-bold text-muted"
+        href={href({
+          widgetSrc: `${widgetAccountId}/widget/app`,
```

Comment:
If the widgetAccountId is a static string I wouldn't it in a variable here. It will not be replaced during runtime.

Reply:
this is not for our widgets, we will use it inside mob's notification widget, it will be passed from props.